### ISSUE TITLE
encoding error in info bar

### ISSUE THREAD

**Issue by hebbet** [armin-pfaeffle/sass-autocompile#8]
in the info bar on successful compile there is an encoding error:

![screen_000066a](https://cloud.githubusercontent.com/assets/1423115/8149998/7cb9681c-12d8-11e5-93c7-184db0f23281.png)

**Comment by armin-pfaeffle** [comment]
Hm... Interesseting... On my Windows computer (Win7, 64bit) it's a dash and not an unknown character.

Which system do you use?

**Comment by hebbet** [comment]
Windows 8.1

**Comment by armin-pfaeffle** [comment]
You can try the following for me:

Open the LESS file `C:\Users\<username>\.atom\packages\sass-autocompile\styles\sass-autocompile.less` and modify line 27 to `content: "\2014";`, so the block looks like:

```
        &::before {
            content: "\2014";
            display: inline-block;
            padding: 0 10px;
            cursor: default;
            color: fadeout(@text-color, 70%);
        }
```

The reload the editor (shortcut: `Ctrl+Alt+R`) and save a SCSS file. Is the bug fixed?

**Comment by hebbet** [comment]
On Mon, Jun 15, 2015 at 12:06 PM, Armin Pfäffle <notifications@github.com>
wrote:

> Is the bug fixed?


yes.

**Comment by armin-pfaeffle** [comment]
Closed with 0.7.1
